### DISCUSSION TITLE
Wings memory problem

### DISCUSSION THREAD

**Discussion by Kaesewuerfel** [pterodactyl/panel#3080] [Support]
Hello,

my wings is going crazy every day and is using up all the Ram/CPU and the server is very slow to respond.

![image](https://user-images.githubusercontent.com/74424677/107161501-5f06df80-699d-11eb-97f6-e0a734e1962f.png)

**Comment by matthewpi** [comment]
Please take a look at https://github.com/pterodactyl/panel/issues/2524#issuecomment-739975666

**Reply by Kaesewuerfel** [reply]
But it's not only the memory. the cpu is going up to 1000 % (10 cores)

**Reply by Kaesewuerfel** [reply]
And again (it isn't cache): 
![image](https://user-images.githubusercontent.com/74424677/107163994-1a834000-69ad-11eb-8808-34f1c69330e7.png)
![image](https://user-images.githubusercontent.com/74424677/107164033-41417680-69ad-11eb-9085-d1343ce45738.png)

**Comment by Kaesewuerfel** [comment]
#push
